src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -499,7 +538,6 @@ public bool ShowPreviewBubbles
             set
             {
                 preferenceSettings.ShowPreviewBubbles = value;
-                showPreviewBubbles = value;
```

Comment:
wasn't this just recently added by @zeusongit ? 
https://github.com/DynamoDS/Dynamo/pull/12025/ 
is it ok to remove?

Reply:
It isn't being used.